examples/MulticeiverDemo/MulticeiverDemo.ino

```diff
@@ -27,13 +27,13 @@ RF24 radio(CE_PIN, CSN_PIN);
 // an identifying device destination
 // Notice that the last byte is the only byte that changes in the last 5
 // addresses. This is a limitation of the nRF24L01 transceiver for pipes 2-5
-// because they use the same first 4 bytes from pipe 1.
-uint64_t address[6] = { 0x7878787878LL,
-                        0xB3B4B5B6F1LL,
-                        0xB3B4B5B6CDLL,
-                        0xB3B4B5B6A3LL,
-                        0xB3B4B5B60FLL,
-                        0xB3B4B5B605LL };
+// because they use the same first 4 MSBytes from pipe 1.
+uint8_t address[6][6] = { { 0x78, 0x78, 0x78, 0x78, 0x78 },
+                          { 0xF1, 0xB6, 0xB5, 0xB4, 0xB3 },
+                          { 0xCD, 0xB6, 0xB5, 0xB4, 0xB3 },
+                          { 0xA3, 0xB6, 0xB5, 0xB4, 0xB3 },
+                          { 0x0F, 0xB6, 0xB5, 0xB4, 0xB3 },
+                          { 0x05, 0xB6, 0xB5, 0xB4, 0xB3 } };

```

Comment:
The increase in available RAM for global variables is from this change.

I switched to using a proper array of buffers to get away from using the older deprecated `openWritingPipe()`. This also is communication with using `memcpy()`.

Reply:
I also fixed the mistaken `address[6][6]` to be `address[6][5]`.